kahuna/public/js/components/gr-image-usage/gr-image-usage.css

```diff
@@ -0,0 +1,35 @@
+.gr-image-usage__field_title {
+    font-weight: bold;
+    color: #999;
+    vertical-align: top;
+    text-align: left;
+}
+
+gr-image-usage-list {
+    display: block;
+}
+
+gr-image-usage-list .date-added {
+    font-size: 11px;
```

Comment:
We could do something [similar to this](https://github.com/guardian/grid/blob/master/kahuna/public/stylesheets/main.css#L325) as we have it in a few places, perhaps `font-tiny`? Not a blocker, maybe a thought for later.